derive-macros/src/lib.rs

```diff
@@ -74,7 +82,16 @@ fn gen_schema_fields(data: &Data) -> TokenStream {
                         _ => panic!("Can only handle <> type path args"),
                     }
                 });
-                quote_spanned! { field.span() => #(#type_path_quoted),* get_struct_field(stringify!(#name))}
+                if have_schema_null {
+                    if let Some(first_ident) = type_path.path.segments.first().map(|seg| &seg.ident) {
+                        if first_ident != "HashMap" && first_ident != "Vec" {
+                            panic!("Can only use schema_container_values_null on HashMap or Vec fields, not {first_ident:?}");
```

Comment:
Why would it be correct to delete null entries from a Vec? It would change the indexes of all entries after it, which could cause a problem if the indexes were somehow meaningful?

(I also wonder what potential use cases that even serves?)